packages/rsocket-core/src/RequestChannelStream.ts

```diff
@@ -496,11 +496,12 @@ export class RequestChannelResponderStream
           return;
         }
 
-        const payload = Reassembler.reassemble(
-          this,
-          frame.data,
-          frame.metadata
-        );
+        const payload = this.hasFragments
+          ? Reassembler.reassemble(this, frame.data, frame.metadata)
+          : {
```

Comment:
Is this addressing a bug that was present or part of Resume support?

Reply:
yes. Likelly